resources/uk/gov/hmcts/helm/check-deprecated-charts.sh

```diff
@@ -16,7 +16,7 @@ deprecationMap["elasticsearch"]="7.17.3"
 for deprecation in "${!deprecationMap[@]}"
 do
   CURRENT_VERSION=$(helm dependency ls charts/${CHART_DIRECTORY}/ | grep "^$deprecation " |awk '{ print $2}' | sed "s/~//g")
-  if [[ -n $CURRENT_VERSION ]] &&  [ $(ver $CURRENT_VERSION) -lt $(ver ${deprecationMap[$deprecation]}) ]; then
+  if [[ -n $CURRENT_VERSION ]] || [ $(ver $CURRENT_VERSION) -lt $(ver ${deprecationMap[$deprecation]}) ]; then
```

Comment:
won't this always mark things as deprecated? 

From what I'm reading it's

if version is provided or version less than deprecation map.

before it was 
if version is provided and the version is less than version in the deprecation map.

(but the bash may have been written wrong)